config-controller/pkg/client/client.go

```diff
@@ -225,17 +226,17 @@ func New(ctx context.Context, opts ...clientOptions) (CachedPolicyClient, error)
 	return &c, nil
 }
 
-func (c *client) ListPolicies(ctx context.Context) ([]*storage.Policy, error) {
-	policies := make([]*storage.Policy, 0, len(c.cache))
-	for _, value := range c.cache {
+func (c *client) ListPolicies(_ context.Context) ([]*storage.Policy, error) {
+	policies := make([]*storage.Policy, 0, len(c.policyObjectCache))
+	for _, value := range c.policyObjectCache {
 		policies = append(policies, value)
 	}
 	return policies, nil
 }
 
-func (c *client) GetPolicy(ctx context.Context, name string) (*storage.Policy, bool, error) {
-	policy, exists := c.cache[name]
-	return policy, exists, nil
+func (c *client) GetPolicy(_ context.Context, name string) (*storage.Policy, bool, error) {
```

Comment:
should we just remove ctx from this method in the interface?

Reply:
We can't because the grpc client needs the ctx

Reply:
There's no grpc calls in GetPolicy, no?

Reply:
we can save this discussion for another time.